### ISSUE TITLE
Alt text

### ISSUE THREAD

**Issue by farishkash** [satyakipatnaik/TeaCozy#3]
`<img src="https://s3.amazonaws.com/codecademy-content/courses/freelance-1/unit-4/img-berryblitz.jpg" style="width:300px;height:200px;">
`
Each image should have a descriptive alt text.  The visually impaired uses screen readers to read this out loud so they can know what the picture is about.
